broker/applications/operators/test/operators.DockerService.spec.js

```diff
@@ -335,15 +335,17 @@ describe('docker-operator', function () {
               uri: `http://${username}:${password}@${docker_url.hostname}`
             });
             mocks.verify();
-          });
+          }); 
       });
       it('returns 201 Created and creates Security Group for shared instance binding', function () {
         let target_space_guid = 'target_id';
         let binding_id = 'binding_id';
         mocks.docker.inspectContainer(instance_id);
-        mocks.cloudController.getSpace(target_space_guid, {
-          'organization_guid': organization_guid
-        });
+        mocks.cloudController.getSpace(target_space_guid,organization_guid
+        // {  
+        //   'organization_guid': organization_guid
+        // }
+        );
```

Comment:
```suggestion
        mocks.cloudController.getSpace(target_space_guid, organization_guid);
```
I feel it is good to remove the comments.